orchestrator/careplanservice/config.go

```diff
@@ -23,5 +26,11 @@ func (c Config) Validate() error {
 	if c.FHIR.BaseURL == "" {
 		return errors.New("careplanservice.fhir.url is not configured")
 	}
+	if c.AuditObserverSystem == "" {
+		return errors.New("careplanservice.auditobserversystem is not configured")
+	}
+	if c.AuditObserverValue == "" {
+		return errors.New("careplanservice.auditobservervalue is not configured")
+	}
```

Comment:
Do we want the URA number here? We can just defer them from the local node (`profile.Identity()`)

Reply:
My thinking is, less configuration options = simpler system. Also, if we ever were to make ORCA multi-tenant, you'd want the tenant (thus, this identity) in the audit i.m.o.